### PR TITLE
add support for named capture of wildcard path

### PR DESCRIPTION
**Pull request by mindplay-dk**

### CODE REVIEW

File: test/test.php [hunk 1]
```diff
@@ -391,6 +391,30 @@ function () use ($router) {
     }
 );
 
+test(
```
Comment (jesperoestergaardjensen) [inline test/test.php:394]: Remember to remove unused test at line 371

File: src/Router.php [hunk 2]
```diff
@@ -119,7 +119,12 @@ protected function match($url)
                 }
 
                 if (isset($current->wildcard)) {
+                    if ($current->wildcard_name) {
+                        $params[$current->wildcard_name] = implode('/', array_slice($parts, $index));
```
Comment (jesperoestergaardjensen) [inline src/Router.php:123]: Is this the only reason $wildcard_name exists?

Comment (mindplay-dk) [reply src/Router.php:123]: Yep

File: test/test.php (hunk 1 above)
Comment (mindplay-dk) [reply test/test.php:394]: This test wasn't unused - there was a bug in `mindplay/testies` which caused tests with duplicate names to override each other. I've fixed this in `mindplay/testies^0.3.1`.

**Pull request merged**